dockerfile/directx/install-components.bat

```diff
@@ -0,0 +1,7 @@
+REM Copyright (c) Microsoft Corporation - All rights reserved
+REM Licensed under the MIT License
+
+start /wait vs_BuildTools.exe --config  %SB_HOME%\dockerfile\directx\mini_vsconfig.json --wait --quiet --norestart > nul
+if %errorlevel% neq 0 (
+  exit /b %errorlevel%
+)
```

Comment:
Why not in dockerfile directly?

Reply:
I tried, but seems if I just put it into dockerfile directly, this command 'start /wait vs_BuildTools.exe --config %SB_HOME%\dockerfile\directx\mini_vsconfig.json --wait --quiet --norestart > nul'  will not wait until finish, just return very fast and run in async

Reply:
@abuccts do you know why?

Reply:
Refer to [vs_buildtools will not correctly install in Docker Windows Container - Developer Community (visualstudio.com)](https://developercommunity.visualstudio.com/t/vs-buildtools-will-not-correctly-install-in-docker/1695778)